### PR TITLE
Validatecontactform

### PR DESCRIPTION
**Pull request by susangoga**
# Dear reviewer:

For this project my partner and I implement a simple validation:📌

- The content of the email field has to be in lowercase.

And we also implement the following interactions:

1.- When the user submits the form, we check if the email is in lowercase.
2.- If the validation is OK, the form is sent.
3.- If the validation is not OK, show an error message to the user near the submit button informing them of the error, and the form is not sent.

Thanks for your feedback👔

### CODE REVIEW

File: navScript.js [hunk 1]
```diff
@@ -13,4 +13,56 @@ function myFunction() {
...
+  //   var txtmail = document.getElementsByName("email");
+  //   var regex = /^([a-z\d\.-]+)@([a-z\d-]+)\.([a-z]{2,3})$/ 
+
+  //   if (!regex.test(txtmail)) {
+  //       document.getElementById("mssg").innerHTML = "Format is not valid";
+  //   } else {
+  //     document.getElementById("mssg").innerHTML = "";
+  //   }
+  
+  // }
```
Comment (GzimAsani) [inline navScript.js]: - [ ] Please remove the commented out code

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @vigehi  and @tamana-Balkhi                  :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with styling looks like the design :+1: 

 ## Required Changes 


- [ ] The [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript)  linters is set correctly project repository, currently you have only html/css linters.
- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: navScript.js (hunk 1 above)
Comment (levy002) [inline navScript.js]: - [ ] As requested by the previous reviewer, Kindly make sure to remove the commented code in your code base.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @susangoga   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] As requested by the previous reviewer, Kindly make sure to set up linters correctly so it runs to check even `es-lint` linter for JavaScript. You can use this[ setup](https://github.com/microverseinc/linters-config/tree/master/html-css-js).


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: navScript.js (hunk 1 above)
Comment (susangoga) [reply navScript.js]: I'm so sorry, I forgot to commit the change, I'm doing it.

thanks a lot!

**Review by Reem-lab — APPROVED** [review]
# Hi Team, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights
- No Linters error ✅
- Followed GitHub flow ✅
- Great working on the functionality  ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**